packages/fxa-content-server/app/scripts/lib/glean/index.ts

```diff
@@ -263,7 +263,7 @@ const recordEventMetric = (eventName: string, properties: EventProperties) => {
       setPasswordThirdPartyAuth.submit.record();
       break;
     case 'set_password_third_party_auth_success':
-      setPasswordThirdPartyAuth.sucess.record();
+      setPasswordThirdPartyAuth.success.record();
```

Comment:
I don't understand why TS wasn't able to help here.  ☹️

Reply:
I also was confused by this, but the import is for a `.js` file so all types are interpretted as 'any'...